docker/Dockerfile.synapse.registration

```diff
@@ -0,0 +1,6 @@
+# Only exists to register a user, temporary solution
```

Comment:
Could you clarify? If I understand correctly, you're building on top of `Dockerfile.synapse.latest` just for the sake of calling

```sh
register_new_matrix_user -c /data/homeserver/yaml -u mjolnir ...
```

right?

Can't you run this as a `RUN` script in `Dockerfile.synapse.latest`?

Alternatively, it would probably be more future-proof if this could be a Node script rather than a Dockerfile script. This node script is something that we'll be able to later extend into a real integration test runner.